pytorch/buildspec-graviton-1-10.yml

```diff
@@ -0,0 +1,39 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK pytorch
+version: &VERSION 1.10.0
+os_version: &OS_VERSION ubuntu20.04
+short_version: &SHORT_VERSION "1.10"
```

Comment:
same as above, it should be 1.12.0

Reply:
This file is for legacy purpose, and not used.